index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="./style.css">
+        <title>Document</title>
```

Comment:
- Please make sure to give your webpage a proper title other than the placeholder text 'Document' as this will help to distinguish your page from other sites open in the browser especially if you have multiple tabs open 🔍.

Reply:
ok thank